global_planner/launch/global_planner_stereo.launch

```diff
@@ -5,7 +5,7 @@
         <arg name="use_three_point_msg" value="$(arg use_three_point_msg)" />
         <arg name="use_stereo_simulation" value="true" />
         <arg name="point_cloud_topic" value="/stereo/points2" />
-        <arg name="model" value="iris_stereo_camera" />
+        <!-- <arg name="model" value="iris_stereo_camera" /> -->
```

Comment:
Even better: change the `global_planner_sitl_mavros.launch` and `mavros_sitl.launch` to use the argument and pass it down. (Currently `mavros_sitl.launch` hardcodes the vehicle to 'iris_depth_camera')